dora/core/common/src/main/java/alluxio/metadata/caching/Cache.java

```diff
@@ -9,10 +9,9 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.master.metastore.caching;
+package alluxio.metadata.caching;
```

Comment:
extract the package from master to common so that dora can use it. 

this is a cache, currently used by master inode store. The cache has a backing store and when new entries added to the cache, it records it and flushes it asynchronously by another thread.